source/js/app.js

```diff
@@ -28,11 +28,13 @@ import KeepInViewPort from './keepInViewPort';
 import ButtonToggle from './buttonToggle';
 import TestimonialCarousel from './testimonials';
 import IframeAcceptance from './iframeAcceptance';
+import Inputs from './inputs';
 import './helpers/swipe';
 
 expandSection();
 setScrollbarCSS();
 IframeAcceptance();
+Inputs();
```

Comment:
Make this name more clear. Microsoft calls this feature "StickyKeys". We should use something similar.

Reply:
I will fix this